R/pkg/R/functions.R

```diff
@@ -2459,6 +2459,7 @@ setMethod("schema_of_csv", signature(x = "characterOrColumn"),
 #' @note from_utc_timestamp since 1.5.0
 setMethod("from_utc_timestamp", signature(y = "Column", x = "character"),
           function(y, x) {
+            .Deprecated("from_utc_timestamp")
```

Comment:
this is actually, incorrect - https://www.rdocumentation.org/packages/base/versions/3.5.3/topics/Deprecated
the string is the "new function"  - instead I think you can try `.Deprecated(msg = "from_utc_timestamp is deprecated for xyz reason")`